taxonomies/ingredients.txt

```diff
@@ -9874,11 +9874,13 @@ sv:Färsk grädde
 en:sour cream, soured cream, grade a cultured cream
 ar:كريمة حامضة
 ca:Crema agra, crema àcida
+da:syrnet fløde
 de:saure Sahne, Sauerrahm
+el:κρέμα γαλακτοσ
 eo:Acidkremo
-es:Crema agria, nata agria
+es:Crema agria, nata agria, nata ácida
 fa:خامه ترش
-fi:Hapankerma
+fi:Hapankerma, ranskankerma
```

Comment:
Ranskankerma = Crème fraîche. I think it's already a synonym for that.

Reply:
Great, I just copied it from ingredient lists. So you see how well producers translate.

Reply:
So ", ranskankerma" should be removed, right?

Reply:
@stephanegigandet Actually, I noticed that ranskankerma was under a wrong ingredient (`en:fresh-cream`) even before aleene's commit. I'll fix it by pushing to this branch.

Reply:
Perfect, thank you @Ban3 !